src/Pim/Component/Catalog/Factory/ProductValue/OptionsProductValueFactory.php

```diff
@@ -104,6 +109,27 @@ protected function checkData(AttributeInterface $attribute, $data)
     }
 
     /**
+     * Returns an ArrayCollection of attribute options.
+     *
+     * @param AttributeInterface $attribute
+     * @param string[]           $data
+     *
+     * @return ArrayCollection
+     */
+    protected function getOptions(AttributeInterface $attribute, array $data)
+    {
+        $options = new ArrayCollection();
```

Comment:
Historical. There is to much work to do on normalizers, including PEF normalizers. Without a green CI, I wouldn't take the risk.
I think it have to be done in another PR.